app/react/translations/list/index.js

```diff
@@ -49,13 +49,11 @@ type PropTypes = {
   breadcrumbPath: Array<string>,
   path: Array<string>,
   push: Function,
-  saveTranslation: Function,
   toggleHierarchy: Function,
-  fillTranslation: Function
 };
 
 type StateTypes = {
-  pressedKeyCode: ?number
+  tabPressed: ?boolean
```

Comment:
Can be `tabPressed` undefined or null? According to code, this property is always declared and boolean type.